docs/changelog/88589.yaml

```diff
@@ -0,0 +1,21 @@
+pr: 88589
+summary: Make composite aggs in datafeeds Generally Available
+area: Machine Learning
+type: feature
+issues: []
+highlight:
+  title: Composite aggregations in datafeeds are Generally Available
+  body: |-
+    The support for 
+    {ml-docs}/ml-configuring-aggregation.html#aggs-using-composite[composite aggregations]
+    in datafeeds is now generally available.
+
+    [discrete]
+    [[early-stopping-dfa]]
+    === Early stopping for data frame analytics
+    Data frame analytics is even faster in 8.4. The new function makes it 
+    possible to stop the process of hyperparameter optimization early in case of 
+    the accuracy gain for a different set of hyperparameter values would be 
+    insignificant. The early stopping of the optimization process results in a 
+    shorter runtime for the data frame analytics job.
```

Comment:
```suggestion
    === Early stopping for data frame analytics
    Data frame analytics is even faster in 8.4. The new function automatically 
    stops the process of hyperparameter optimization early in case of the 
    accuracy gain for a different set of hyperparameter values would be 
    insignificant. The early stopping of the optimization process results in a 
    shorter runtime for the data frame analytics job.
```